### PR TITLE
fix: mobile schedule menu

### LINKED ISSUE #5919: Mobile schedule: Fix menu
**Current State:**
Screenshot_20201204_230526_com android chrome

Please change it as follows:
**1. Version with not expanded search**
   1. Move the menu as follows
   2. Reduce the space to the schedule (also ensure there is not too much space if there are tracks)

101238106-2b2de600-36de-11eb-8637-7c46f38eadb4

**2. When the user clicks the search button, expand the search bar as in the below screenshot.**

101238106-2b2de600-36de-11eb-8637-7c46f38eadb4-a

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #5919 

#### Short description of what this resolves:
Search bar for mobile schedule enhanced

#### Changes proposed in this pull request:

- Expandable search bar implemented

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/styles/libs/_helpers.scss [hunk 1]
```diff
@@ -98,3 +98,15 @@ $spacer-heights: 50 100 200 300 400 500 600 700 800 900;
 .no-wrap {
   white-space: nowrap;
 }
+
+.search-bar {
```
Comment (daretobedifferent18) [inline app/styles/libs/_helpers.scss:102]: This class is already defined

Comment (daretobedifferent18) [reply app/styles/libs/_helpers.scss:102]: Either use diff class name

Comment (daretobedifferent18) [reply app/styles/libs/_helpers.scss:102]: it is breaking other styling. Pls revert

File: app/styles/pages/events.scss [hunk 2]
```diff
@@ -20,7 +20,6 @@
   border-radius: .5rem;
   color: #808080;
 
-
```
Comment (daretobedifferent18) [inline app/styles/pages/events.scss:23]: revert

File: app/styles/libs/_helpers.scss (hunk 1 above)
Comment (nlok5923) [reply app/styles/libs/_helpers.scss]: changed class name.

**Review by daretobedifferent18 — COMMENTED** [review]
Screenshot from 2020-12-15 22-48-29
Does not looks good

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
new review today

your result is not matching with desired result.

i can give you some suggestions for solving this.
use flex-box.
put search box, timezone, box and sort by drop down into a flex box.
after this they will remain in same row.
at mobile view when search box is active and expanded, hide timezone and sort by dropbox. 

You can check desired result.
https://user-images.githubusercontent.com/1583873/101238471-d0e25480-36e0-11eb-9994-14942ecef405.jpg
https://user-images.githubusercontent.com/1583873/101238448-9e385c00-36e0-11eb-9f6a-83ad640c5143.jpg

**Comment by nlok5923** [comment]
@sachinchauhan2889 Thanks for suggesting please can tell me how can i reduce time zone picker width.

**Comment by sachinchauhan2889** [comment]
Put it inside a div and use width property to reduce width

**Comment by iamareebjamal** [comment]
@nlok5923 Please complete the PR or close it

**Pull request closed without merging**